tests/models/test_dagbag.py

```diff
@@ -44,19 +44,20 @@
 from tests.test_utils import db
 from tests.test_utils.asserts import assert_queries_count
 from tests.test_utils.config import conf_vars
-from tests.test_utils.permissions import delete_dag_specific_permissions
 
 
 class TestDagBag:
     @classmethod
     def setup_class(cls):
         cls.empty_dir = mkdtemp()
+        db.clear_dag_specific_permissions()
 
     @classmethod
     def teardown_class(cls):
         shutil.rmtree(cls.empty_dir)
+        db.clear_dag_specific_permissions()
 
-    def setup_methods(self) -> None:
+    def setup_method(self) -> None:
         db.clear_db_dags()
         db.clear_db_runs()
         db.clear_db_serialized_dags()
```

Comment:
I actually meant this entire function can be merged into `setup_class`. Sorry for not being clear. The current change is OK if we’re to minimise the changed lines, although I’d still prefer we changing this to make the test suite ever so slightly faster.

Reply:
Fixed, any better?

Reply:
Oops, now it’s not cleaning up enough. Since the tests trigger DAGs, cleanup should still happen in `teardown_method`, not `teardown_class`. This way, cleanup happens once before any tests a run (`setup_class`), and once after _each_ test (`teardown_method`).

Reply:
I looked into it and looks like cleaning after each method is not required. I have added where it's necessary to do that before starting to test and final teardown_class will clean up after once all complete. 
I have run the tests and all passes with this setup.